src/SumAverage.java

```diff
@@ -34,8 +40,15 @@ public static int sum(int[] numbers) {
    * @return average of integers
    */
   public static double average(int lowerBound, int upperBound) {
-    // TODO fill in code here using while loop and replace the return statement
-    return -1;
+    int i = lowerBound;
+    int sum = 0;
+    int count = 0;
```

Comment:
minor comment: `count` variable is not necessary

Reply:
fixed